### PR TITLE
Add sqrt stretch to image plot

### PR DESCRIPTION
**Pull request by facero**
Using from astropy.visualization import simple_norm 
to be able to plot sqrt, linear or log images in the image.plot function

### CODE REVIEW

**Comment by cdeil** [comment]
@adonath - I feel the same way.

One improvement would be to mention and link to the Astropy docs that talk about `simple_norm`, so that users looking at this function see how they can fully control the norm, or to add one example in an `Examples` section at the end of the docstring.

@adonath - Please decide and if you agree improve the docstring and merge this PR.

**Comment by facero** [comment]
@adonath in the way it is right now it is not fully flexible as the user can not specify a user defined norm otherwise imshow() receives two parameters named norm. I tried it and get this error:
`imshow() got multiple values for keyword argument 'norm' `

I can add a simple check for that and add some links to astropy.visualizatoin in the docstring.

**Comment by adonath** [comment]
@facero This issue could be handled in one line using:
```python
kwargs.set_default('norm', norm)
```
I agree with @cdeil, that extending the docs with a least one example for `simple_norm` would be great, so please go ahead. Once the suggestions have been addressed, this PR is ready to merge from my side

**Comment by facero** [comment]
Thanks @adonath for the kwargs.setdefault('norm', norm) idea. This is what I needed. 
I've added this, one example and url link in docstring.

File: gammapy/image/core.py [hunk 1]
```diff
@@ -888,7 +888,13 @@ def plot(self, ax=None, fig=None, add_cbar=False, stretch='linear', **kwargs):
         fig : `~matplotlib.figure.Figure`, optional
             Figure
         stretch : str, optional
-            Scaling for image ('linear', 'sqrt', 'log')
+            Scaling for image ('linear', 'sqrt', 'log').
+            Similar to normalize and stretch functions in ds9.
```
Comment (adonath) [inline gammapy/image/core.py]: @facero Thanks for adding this example! Just one additional comment:

Could you move this example into a new "examples" sections of the `SkyImage.plot()` docstring? 
This has the advantage that the code will be formatted nicely and users can directly copy and paste from the docs. (see [here](https://github.com/gammapy/gammapy/blob/master/gammapy/image/core.py#L1024) how this section is defined...). Ideally the example you give should work, when just copied and pasted in a notebook or IPython shell. Maybe you can extend the example to achieve this. You could e.g. use a Fermi-LAT example image (as done  [here](https://github.com/gammapy/gammapy/blob/master/docs/image/index.rst#getting-started)).

Comment (facero) [reply gammapy/image/core.py]: @adonath Done. 
I added a standalone example. Tested it and it worked for me. Let me know if this is ok.

**Comment by adonath** [comment]
Merged. Thanks @facero!

**Pull request merged**